docs/design/design-reviews/recipes/preferred-diagram-tooling.md

```diff
@@ -1,15 +1,30 @@
 # Preferred Diagram Tooling
 
 At each stage in the engagement process, diagrams are a key part of the design review.
-The preferred tooling for creating and maintaining diagrams is the `.drawio` format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
+The preferred tooling for creating and maintaining diagrams is to choose **one** of the following:
 
-The `.drawio` format uses the metadata layer within the PNG file-format to hide SVG vector graphics representation, then renders the .png when saving.
-This clever use of both the meta layer and image layer allows anyone to further edit the PNG file.
-It also renders like a normal PNG in browsers and other viewers, making it easy to transfer and embed.
-Furthermore, it can be edited within VSCode very easily using the [Draw.io Integration VSCode Extension](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).
+- Microsoft Visio
```

Comment:
NIT: perhaps add links to Visio and Power point?

Reply:
Thanks Shiran. 

I hesitated. Then I thought readers of this content would know where to find Visio or PowerPoint, while they might discover drawio tools.

This does not seem to be a strong requirement on your side either. I'll resolve the conversation as is.

PS: The links that would last quite long would probably have been <https://microsoft.com/visio> and <https://microsoft.com/powerpoint>.